### PR TITLE
Journalist can create article

### PR DESCRIPTION
**Pull request by DIzosimov**
[PT Link](https://www.pivotaltracker.com/story/show/169253737)

## PR Content
- Create Article Component
- POST axios path in module for article creation
- Happy and sad path test for creating article
- Fixture files for tests

## Screenshots
Screenshot 2019-10-30 at 16 42 34

### CODE REVIEW

**Review by miyeaier — APPROVED** [review]

File: cypress/integration/journalistCanCreateArticle.spec.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    cy.get('#create-article').click()
+
+    cy.get('#article-form').within(() => {
+      cy.get('#title-input').type('Trump has gone insane')
+      cy.get('#content-input').type('Trump has been diagnosed with crazy syndrome')
+      cy.get('#author-input').type('Faraz')
+      cy.get('#category-input').should('contain', '')
+      
+      cy.get('#submit-article').click()
+
```
Comment (piavW) [inline cypress/integration/journalistCanCreateArticle.spec.js]: remove extra line

File: src/Components/ListArticles.jsx [hunk 2]
```diff
@@ -47,16 +47,16 @@ class ListArticles extends Component {
       renderListArticles = (
         <>
           {articleData.data.map(art => {
+            debugger
```
Comment (piavW) [inline src/Components/ListArticles.jsx]: remove debugger

File: src/Modules/RequestArticles.js [hunk 3]
```diff
@@ -14,4 +14,22 @@ const getData = async () => {
   }
 }
 
-export { getData }
\ No newline at end of file
+const submitArticle = async (title, content, author, category, publish_date) => { //add <- publish_date here %d %B %Y (30 October 2019)
```
Comment (piavW) [inline src/Modules/RequestArticles.js]: comment?

**Review by piavW — CHANGES_REQUESTED** [review]
See comments

**Review by piavW — APPROVED** [review]

**Review by beccaburns — APPROVED** [review]

**Review by yasmineezequiel — APPROVED** [review]

**Review by oliverochman — APPROVED** [review]

**Pull request merged**